### PR TITLE
fixes #560

### LINKED ISSUE #560: testing error: modules/3d_image_transforms.ipynb
**Describe the bug**
```
Running ./modules/3d_image_transforms.ipynb
Checking PEP8 compliance...
Running notebook...
/opt/conda/lib/python3.8/site-packages/papermill/iorw.py:58: FutureWarning: pyarrow.HadoopFileSystem is deprecated as of 2.0.0, please use pyarrow.fs.HadoopFileSystem instead.
  from pyarrow import HadoopFileSystem

Executing:   0%|          | 0/44 [00:00<?, ?cell/s]
Executing:   2%|▏         | 1/44 [00:01<01:03,  1.47s/cell]
Executing:   7%|▋         | 3/44 [00:05<01:23,  2.03s/cell]
Executing:   9%|▉         | 4/44 [00:08<01:31,  2.28s/cell]
Executing:  23%|██▎       | 10/44 [01:22<05:33,  9.80s/cell]
Executing:  41%|████      | 18/44 [01:23<01:45,  4.07s/cell]
Executing:  48%|████▊     | 21/44 [01:23<01:12,  3.13s/cell]
Executing:  50%|█████     | 22/44 [01:23<01:02,  2.84s/cell]
Executing:  64%|██████▎   | 28/44 [01:24<00:24,  1.54s/cell]
Executing:  66%|██████▌   | 29/44 [01:24<00:21,  1.42s/cell]
Executing:  66%|██████▌   | 29/44 [01:25<00:44,  2.94s/cell]
Traceback (most recent call last):
  File "/opt/conda/bin/papermill", line 8, in <module>
    sys.exit(papermill())
  File "/opt/conda/lib/python3.8/site-packages/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
  File "/opt/conda/lib/python3.8/site-packages/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
  File "/opt/conda/lib/python3.8/site-packages/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/conda/lib/python3.8/site-packages/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "/opt/conda/lib/python3.8/site-packages/click/decorators.py", line 26, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/opt/conda/lib/python3.8/site-packages/papermill/cli.py", line 250, in papermill
    execute_notebook(
  File "/opt/conda/lib/python3.8/site-packages/papermill/execute.py", line 122, in execute_notebook
    raise_for_execution_errors(nb, output_path)
  File "/opt/conda/lib/python3.8/site-packages/papermill/execute.py", line 234, in raise_for_execution_errors
    raise error
papermill.exceptions.PapermillExecutionError: 
---------------------------------------------------------------------------
Exception encountered at "In [16]":
---------------------------------------------------------------------------
IndexError                                Traceback (most recent call last)
/tmp/ipykernel_12398/3242378529.py in <module>
      3 plt.subplot(1, 2, 1)
      4 plt.title("image")
----> 5 plt.imshow(image[0, :, :, 30], cmap="gray")
      6 plt.subplot(1, 2, 2)
      7 plt.title("label")

IndexError: too many indices for array: array is 3-dimensional, but 4 were indexed
```

### PR DESCRIPTION
**Pull request by wyli**
Signed-off-by: Wenqi Li <wenqil@nvidia.com>

Fixes #560

### Description
follow up of https://github.com/Project-MONAI/tutorials/pull/546
### Status
**Ready**

### Checks

- [ ] Notebook runs automatically `./runner [-p <regex_pattern>]`

### CODE REVIEW

File: modules/3d_image_transforms.ipynb [hunk 1]
```diff
@@ -277,27 +258,11 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 8,
+   "execution_count": null,
    "metadata": {
     "tags": []
    },
-   "outputs": [
```
Comment (Nic-Ma) [inline modules/3d_image_transforms.ipynb]: Maybe we should keep the output example?

Thanks.

File: modules/3d_image_transforms.ipynb [hunk 2]
```diff
@@ -330,23 +295,11 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
+   "execution_count": null,
    "metadata": {
     "tags": []
    },
-   "outputs": [
```
Comment (Nic-Ma) [inline modules/3d_image_transforms.ipynb]: Maybe we should keep the output example?

Thanks.

File: modules/3d_image_transforms.ipynb [hunk 3]
```diff
@@ -457,23 +410,23 @@
...
+      "[[ 0.0000000e+00 -9.7656202e-01  0.0000000e+00  4.7683716e-07]\n",
+      " [-9.7656202e-01  0.0000000e+00  0.0000000e+00  4.7683716e-07]\n",
+      " [ 0.0000000e+00  0.0000000e+00 -5.0000000e+00  2.7000000e+02]\n",
+      " [ 0.0000000e+00  0.0000000e+00  0.0000000e+00  1.0000000e+00]]\n"
      ]
     }
    ],
    "source": [
-    "data_dict = orientation(data_dict)\n",
+    "data_dict = orientation(datac_dict)\n",
```
Comment (Nic-Ma) [inline modules/3d_image_transforms.ipynb]: Is this a typo?

Thanks.

**Review by Nic-Ma — APPROVED** [review]
Thanks for the quick fix, seems there are much change in this PR, may I know which is the key change to fix the error? Or what's the root cause?

Thanks.

File: modules/3d_image_transforms.ipynb (hunk 3 above)
Comment (wyli) [reply modules/3d_image_transforms.ipynb:455]: it's not a typo, previously I swap the cells but forgot to change the variables. now it's tested with v0.8.1. I think we are good for tagging for tutorial 0.8.1 after this PR? do you have any concerns?

**Pull request merged**